presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -365,6 +365,19 @@ public boolean isLimitGuaranteed(ConnectorSession session)
         return true;
     }
 
+    @Override
+    public void setColumnComment(JdbcIdentity identity, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
+    {
+        String sql = format(
+                "ALTER TABLE %s MODIFY %s %s(%s) COMMENT %s",
+                quoted(handle.getRemoteTableName()),
+                column.getColumnName(),
+                column.getJdbcTypeHandle().getJdbcTypeName().get(),
+                column.getJdbcTypeHandle().getColumnSize(),
+                format("'%s'", comment.get()));
```

Comment:
This logic will remove some existing column information (e.g. `DEFAULT`). We should keep such information.

Reply:
@ebyhr Yes, I missed on this. Currently with this command, it's overriding existing values. The same command with original column information should help in retaining existing column information. Though I could find only a nullable check from existing objects but not any other column info, could you please help me with your input, what logic can I use here to get the column original definition?

Reply:
You will be able to get the column definition from JDBC `DatabaseMetaData.getColumns`.